zerver/webhooks/sentry/view.py

```diff
@@ -81,6 +81,20 @@
 }
 
 
+def is_sample_event(event: Dict[str, Any]) -> bool:
+    # These are just some workarounds; there is no documented way to check
+    # if an event is a sample event or not.
+    tags = event.get("tags", [])
+    if ["sample_event", "yes"] in tags:
+        return True
+    title = event.get("title", "")
+    if title == "This is an example Python exception":
+        return True
+    # This happens if someday Sentry decides to change the title of the
+    # sample event or remove the [sample_event, yes] tag.
+    return False  # nocoverage
```

Comment:
Doesn't this nocoverage line means that we don't test with any of the old-style Raven data?  Can we add a fixture and test for a non-sample-event one of those?

Reply:
Agreed. We should add an old-style test to the integration.